solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/EmptyView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  EmptyView.swift
+//  DeliveryApp
+//
+//  Created by Sara Batista dos Santos Felix (P) on 22/02/22.
+//
+
+import UIKit
+
+final class EmptyView: UIView {
+    private lazy var emptyLabel: UILabel = {
+        let emptyTextLabel = UILabel()
+        emptyTextLabel.translatesAutoresizingMaskIntoConstraints = false
+        emptyTextLabel.text = "Endereço não encontrado"
+        emptyTextLabel.font = .boldSystemFont(ofSize: 10)
+        emptyTextLabel.textAlignment = .center
+        return emptyTextLabel
+    }()
+    
+    private lazy var emptyText: UITextField = {
+        let emptyTextField = UITextField()
+        emptyTextField.translatesAutoresizingMaskIntoConstraints = false
+        emptyTextField.placeholder = "Procure por ruas com número e bairro utilizando o campo de busca"
+        emptyTextField.textAlignment = .center
+        return emptyTextField
+    }()
```

Comment:
Aqui a mesma coisa, e também, ao invés de utilizar um `UITextField`, considere utilizar uma `UILabel` aplicando as formatações necessárias para parecer um subtítulo.
```suggestion
    private lazy var subtitleLabel: UILabel = {
        let subtitleLabel = UILabel()
        subtitleLabel.translatesAutoresizingMaskIntoConstraints = false
        subtitleLabel.text = "Procure por ruas com número e bairro utilizando o campo de busca"
        titleLabel.font = .subtitle
        subtitleLabel.textAlignment = .center
        return subtitleLabel
    }()
```